### DISCUSSION TITLE
Error: 500: failed to create user account

### DISCUSSION THREAD

**Discussion by joekman990** [AntlerVC/firetable#923] [Support / Q&A] [answered]
👉 **Please describe how to reproduce this bug here.**
i had deleted a project and then tried to create a new project with the same firebase project and when it was created it sent me this error when i went back into my firestore and deleted the _rowy_ collection the error went away

**Comment by harinij** [comment, marked as answer]
@joekman990 Thanks for pointing this out, we will into fixing this flow.

**Reply by BeeBombshell** [reply]
Hey @joekman990, this issue has now been resolved from our end.
